### DISCUSSION TITLE
Adding Biosynthetic Gene Clusters (BGC) prediction

### DISCUSSION THREAD

**Discussion by camilogarciabotero** [oschwengers/bakta#53] [Ideas]
Hi all

Thanks for working on this fantastic project! I was wondering if there is already, in the state of art, a genome annotator that includes (or may include) annotation of biosynthetic gene clusters (BGCs). And I thought that maybe the compatibility of Inputs and Outputs is a delicate matter or other issues related to scalability and speed... Anyway, since I haven't found one yet, I thought it could be great to add this feature to `bakta` because its efficiency and then came across with [GECCO](https://github.com/zellerlab/GECCO) which sounds like a pretty good alternative to include in the annotation pipeline of `bakta` 

Just an idea,
Cheers...
Camilo

**Comment by oschwengers** [comment]
Hi @camilogarciabotero thanks for bringing this up. BGCs and GECCO would definitely be an interesting option. I'll keep an eye on the upcoming publication and see what the runtime specs are as I'm keen to keep overall runtime for Bakta rather low.
